lib/workload/stateless/stacks/filemanager/deploy/stack.ts

```diff
@@ -153,12 +153,14 @@ export class Filemanager extends Stack {
     new HttpRoute(this, 'PatchHttpRoute', {
       httpApi: httpApi,
       integration: apiIntegration,
+      authorizer: apiGateway.cognitoAdminGroupAuthorizer,
```

Comment:
I think the FM might be a use case where only the admin user (including `orcabus_token`) could do non-read methods (e.g. POST, PATCH). @mmalenic

Reply:
@victorskl Just an important note on each stage we must put the `orcabus_token` service on the admin Cognito group.

Reply:
Got it, will do. Thanks, Will.